source/diagnostics.tex

```diff
@@ -2447,10 +2447,12 @@
 The semantics of this function are \impldef{semantics of \tcode{breakpoint}}.
 
 \begin{note}
-When invoked, the execution of the program temporarily halts and execution is
-handed to the debugger until such a time as: The program is terminated by the
-debugger, or the debugger resumes execution of the program as if the function
-was not invoked.
+It is intended that, when invoked with a debugger present, the execution of the
+program temporarily halts and execution is handed to the debugger until the
+program is either terminated by the debugger or the debugger resumes execution
+of the program as if the function was not invoked. In particular, there is no
+intent for a call to this function to accomodate resumption of the program in a
+different manner. If there is no debugger present, the program can abend.
```

Comment:
@Dani-Hub, how about:
```suggestion
different manner. If there is no debugger present, execution of the program can
end abnormally.
```
?

Reply:
Looks like a great improvement to may - Thanks Hubert!